src/main/java/agh/ics/oop/BoundingRect.java

```diff
@@ -0,0 +1,10 @@
+package agh.ics.oop;
+
+public class BoundingRect {
+    public Vector2d lowerLeftCorner;
+    public Vector2d upperRightCorner;
```

Comment:
publiczne atrybuty = zło. Ta klasa to idealny materiał na javowy rekord, wtedy kodu będzie jeszcze mniej, a hermetyzację zachowamy :)